core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.ts

```diff
@@ -91,6 +92,10 @@ export class OpportunitiesListItemComponent {
           this.inReviewProgressStyle = {
             width: inReviewTranslationsPercentage + '%'
           };
+          this.isOpportunityButtonDisabled = (
```

Comment:
Do not start a variable name with a verb. Maybe opportunityButtonDisabled? (The other variable names starting with `is` are not expected / wrong)

Reply:
Done